portfolio/src/main/webapp/style.css

```diff
@@ -115,14 +116,24 @@ body {
   list-style: none;
 }
 
+/* Styling the Comments Section */
+#commentsTextArea {
```

Comment:
nit: since it looks like you're using camelCase everywhere else, rename `.speech-bubble` to `.speechBubble` for consistency